calico/getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -220,9 +220,9 @@ The VPP dataplane has the following requirements:
 - A blank Kubernetes cluster, where no CNI was ever configured.
 - These [base requirements]({{site.baseurl}}/getting-started/kubernetes/requirements), except those related to the management of `cali*`, `tunl*` and `vxlan.calico` interfaces.
 
-> **Note**: If using `kubeadm` to create the cluster please make sure to specify the pod network CIDR using `--pod-network-cidr` command-line argument, for example, `sudo kubeadm init --pod-network-cidr=192.168.0.0/16`. If 192.168.0.0/16 is already in use within your network you must select a different pod network CIDR.
+   > **Note**: If using `kubeadm` to create the cluster please make sure to specify the pod network CIDR using `--pod-network-cidr` command-line argument, ie, `sudo kubeadm init --pod-network-cidr=192.168.0.0/16`. If 192.168.0.0/16 is already in use within your network you must select a different pod network CIDR.
```

Comment:
ie should be "i.e.,`

Reply:
Although I'm not sure we really need this change.

If anything, I might do this (assuming it renders OK):

> command-line argument. For example:
> ```
> sudo kubeadm init ...
> ```

Reply:
> ie should be "i.e.,`

Thanks. Corrected.

Reply:
> Although I'm not sure we really need this change.
> 
> If anything, I might do this (assuming it renders OK):
> 
> > command-line argument. For example:
> > ```
> > sudo kubeadm init ...
> > ```

Tried it out and I am not entirely satisfied with the rendering. I am not too sure it looks great - the grey box containing "sudo kubeadm init.." inside the blue "Note" box. The alignment is not proper. So I am leaving it unchanged. Hope thats ok.

Reply:
Lastly, an earlier version of the commit seems to have made it into master as part of the following commit:

> commit 0c61aab756627c15436dd01ef275542a6ad5d98d
> Merge: eb8645469 e80b1f242
> Author: Casey Davenport <caseydavenport@users.noreply.github.com>
> Date:   Fri Apr 29 09:29:29 2022 -0700
> 
>     Merge pull request #5995 from ctauchen/netlify-build-test
> 
>     Duplicate to #5985
> 
> 
> 
> commit e80b1f2429c0651ded4c2d1d2573e498790cd46f
> Author: Christopher <chris.tauchen@tigera.io>
> Date:   Fri Apr 29 15:10:47 2022 +0100
> 
>     Duplicate to #5985

    
I don't know how it happened :)

But the problem with the above commit is it does not have my original commit msg so hopefully this PR will amend that.